src/cascadia/TerminalSettingsEditor/MainPage.cpp

```diff
@@ -256,6 +258,17 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         {
             contentFrame().Navigate(xaml_typename<Editor::Rendering>(), winrt::make<RenderingPageNavigationState>(_settingsClone.GlobalSettings()));
         }
+        else if (clickedItemTag == actionsTag)
+        {
+            auto actionsState{ winrt::make<ActionsPageNavigationState>(_settingsClone) };
+            actionsState.OpenJson([weakThis = get_weak()](auto&&, auto&& arg) {
+                if (auto self{ weakThis.get() })
+                {
+                    self->_OpenJsonHandlers(nullptr, arg);

```

Comment:
Since we're binding our `_OpenJsonHandlers`, we should make sure we use the same tooltip over there too.

Reply:
![image](https://user-images.githubusercontent.com/18356694/108913261-56deaf00-75ef-11eb-93ab-753bee7a4899.png)

Reply:
(yes I'm adding the alt thing)